Makefile

```diff
@@ -2,7 +2,6 @@
 
 init:
 	pip install pipenv
-	pipenv lock
```

Comment:
Sorry, are you asking if the removal is necessary, or the locking?

We can leave this in, but it's an extra 4-5 minutes of CI time that doesn't actually need to run.  Pipenv is no longer worrying about hashes on a project specific level, so platform conflicts aren't a worry now. If there is an actual change to the Pipfile not represented, Pipenv will rebuild things without this command, otherwise it uses the current Pipfile.lock.

Like I said, we can not include this commit here, but this line should be removed at some point.